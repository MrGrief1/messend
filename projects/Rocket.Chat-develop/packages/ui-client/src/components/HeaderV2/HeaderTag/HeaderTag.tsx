import { Box, Tag } from '@rocket.chat/fuselage';
import type { ComponentProps } from 'react';

type HeaderTagProps = ComponentProps<typeof Tag>;

const HeaderTag = ({ children, ...props }: HeaderTagProps) => (
	<Box p={4} withTruncatedText minWidth='x32'>
		<Tag medium {...props}>
			{children}
		</Tag>
	</Box>
);

export default HeaderTag;
