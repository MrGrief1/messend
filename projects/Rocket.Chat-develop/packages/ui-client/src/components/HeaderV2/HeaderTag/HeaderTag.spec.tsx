import { render } from '@testing-library/react';

import HeaderTag from './HeaderTag';

it('should match snapshot', () => {
	const { baseElement } = render(<HeaderTag>Test Tag</HeaderTag>);
	expect(baseElement).toMatchSnapshot();
});
