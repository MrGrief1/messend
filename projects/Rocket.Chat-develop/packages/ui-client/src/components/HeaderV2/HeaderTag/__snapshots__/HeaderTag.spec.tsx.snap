// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`should match snapshot 1`] = `
<body>
  <div>
    <div
      class="rcx-box rcx-box--full rcx-css-1kopyx3"
    >
      <span
        class="rcx-box rcx-box--full rcx-tag rcx-tag--medium"
      >
        <span
          class="rcx-tag__inner"
        >
          Test Tag
        </span>
      </span>
    </div>
  </div>
</body>
`;
